pkg/controller/stacks/stack/stack.go

```diff
@@ -707,20 +708,8 @@ func (h *stackHandler) prepareHostAwareDeployment(d *apps.Deployment, tokenSecre
 	d.Name = o.Name
 	d.Namespace = o.Namespace
 
-	return nil
-}
-func (h *stackHandler) prepareHostAwareJob(j *batch.Job, tokenSecret string) error {
```

Comment:
Was this just not being used anywhere? The only other references I'm seeing are in tests.

Reply:
In a resolved comment @jbw976 pointed out that this function was in support of `job` based Stack controllers, which we removed support for in a past release.